container/controller/controller/runtime.py

```diff
@@ -107,11 +125,14 @@ class KubeDomainRuntime():
   selector:
     app: compute
     domain: {DOMNAME}
+  type: NodePort
   ports:
   - name: libvirt
     port: 16509
+    nodePort: 30001
```

Comment:
Why do you set a static port here and below?

Can't we use the dyn allocated ones?